.github/bin/publish-binaries

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+#
+# Upload binary artifacts when a new release is made.
+#
+# We should be invoked with a pattern for files to upload,
+# and optionally the release-ID to associate them with.
+#
+# These will be stored in the environment as:
+#
+#   INPUT_ARGS
+#
+#   INPUT_RELEASEID
+#
+# Taken from https://github.com/skx/github-action-publish-binaries
+# Action that depends on Docker which works only on Linux
```

Comment:
I guess this is why you inlined the script ? Maybe could you says this also in the commit message to explain the why of this commit.

Also, did you try to find alternative GA to publish release artifacts ? For instance, there is https://github.com/svenstaro/upload-release-action (i didn't check if it is a good one).

Reply:
Switched to [another one](https://github.com/softprops/action-gh-release) that is slightly more followed / better as it creates release on tag creation